client/my-sites/site-settings/settings-jetpack/main.jsx

```diff
@@ -27,8 +27,7 @@ const SiteSettingsJetpack = ( {
 	action,
 	translate,
 } ) => {
-	//todo: this check makes sense in Jetpack section?
-	if ( ! siteIsJetpack ) {
+	if ( ! siteIsJetpack || site.is_wpcom_atomic ) {
```

Comment:
This condition is correct as per the specs in the PR description, but it looks odd at first sight. First thing that comes to my mind is why the markup below should be rendered if the site is AT and how does it related to the `! siteIsJetPack` expression. The following is not strictly needed for merging this PR, but I'll leave it here as a potential idea for improvement.

I'd wrap this in a helper method to make it easier to read and understand. I'd also add a comment explaining why an AT site is part of that expression:

```js
(...)
	// Jetpcack settings for AT sites are abstracted away from the user
	const cannotConfigureJetpack = ! siteIsJetpack || site.is_wpcom_atomic;

	//todo: this check makes sense in Jetpack section?
	if ( cannotConfigureJetpack ) {
(...)
```

Notice how reading the condition feels much more natural now.

Reply:
Done!